src/planwise/endpoint/projects.clj

```diff
@@ -11,5 +11,10 @@
       (let [projects (projects/list-projects service)]
         (response projects)))
 
+    (GET "/:id/details" [id]
```

Comment:
`/details` is required to avoid clashing with `/project/:id`, which is captured by the home endpoint. Should we move all json requests under an `/api/` context? Or route based on content-type? Or add a `.json` extension to all these endpoints?